modules/leftist-heap/include/leftist_heap_app.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 Alexandra Tyurmina
+
+#ifndef MODULES_LEFTIST_HEAP_INCLUDE_LEFTIST_HEAP_APP_H_
+#define MODULES_LEFTIST_HEAP_INCLUDE_LEFTIST_HEAP_APP_H_
+
+#include <string>
+#include "include/leftist_heap.h"
+
+class HeapSort {
+ public:
+  std::string operator()(int argc, const char** argv);
+ private:
+  std::string message_;
+  void Info(const char* appname, const char* message = "");
+  bool CheckArguments(int argc, const char** argv);
```

Comment:
Лучше поменять местами private и public

```suggestion
 private:
  std::string message_;
  void Info(const char* appname, const char* message = "");
  bool CheckArguments(int argc, const char** argv);
  
 public:
  std::string operator()(int argc, const char** argv);
```